Sources/ImgixSwift-Umbrella-Header.h

```diff
@@ -5,5 +5,3 @@
 //  Created by Paul Straw on 7/5/16.
 //
 //
-
-#import "NSString+ImgixSwiftMd5.h"
```

Comment:
If we remove all C/ObjC files, are we still required to provide the `ImgixSwift-Umbrella-Header.h` and `ImgixSwift.modulemap` files?

Reply:
After some more searching (pasted below) I believe the answer to my question is no. I was a bit confused by a build error after removing the umbrella header and modulemap files. Luckily, I was able to resolve it by clearing the module file configuration under the project `Build Settings` as shown below:

<img width="603" alt="Screen Shot 2020-02-03 at 3 18 50 AM" src="https://user-images.githubusercontent.com/15919091/73649183-13a44d80-4634-11ea-991f-93d46622c813.png">

* [Modular framework, creating and using them](http://nsomar.com/modular-framework-creating-and-using-them/)
* [It Looks Like You’re Still Trying to Use a Framework](https://www.bignerdranch.com/blog/it-looks-like-youre-still-trying-to-use-a-framework/)